deploy.Jenkinsfile

```diff
@@ -0,0 +1,97 @@
+def deployNode (host) {
+    return {
+         sshagent (credentials: ['buildagent-matcher']) {
+             sh "ssh -o StrictHostKeyChecking=no -l buildagent-matcher ${host} hostname"
+
+             sh "scp ./waves/node/target/waves-devnet*all*.deb buildagent-matcher@${host}:/home/buildagent-matcher"
+             sh "scp ./matcher/dex-load/src/main/resources/reinstallNode.sh buildagent-matcher@${host}:/home/buildagent-matcher"
+             sleep time: 80000, unit: 'MILLISECONDS'
```

Comment:
Why?

Reply:
unfortunately, jenkins doesn't wait for scp command finished... so we need to wait for sure that shell script will be success